provider/workspace_preset.go

```diff
@@ -119,9 +135,82 @@ func workspacePresetDataSource() *schema.Resource {
 								},
 							},
 						},
+						"autoscaling": {
+							Type:        schema.TypeList,
+							Description: "Configuration block that defines autoscaling behavior for prebuilds. Use this to automatically adjust the number of prebuild instances based on a schedule.",
+							Optional:    true,
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"timezone": {
```

Comment:
I love that we explicitly don't have a default value here 👍 nice.

Reply:
I recommend defaulting to `TZ` or `/etc/localtime`, and falling back to UTC. There's a helper function `tz.TimezoneIANA` for this.